src/popup/Popup.tsx

```
@@ -35,7 +25,28 @@ export const Popup = () => {
     setCurrentBlock,
     blockInterval,
     configData,
-  } = context;
+  } = useSharedState();
+
+  useEffect(() => {
```

ESLint

[error] Expected to return a value at the end of arrow function.

(consistent-return)